src/components/MerchantsPage/GiftMealHighlightBox.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import Modal from '../Modal';
+import { SET_MODAL_VIEW } from '../../utilities/hooks/ModalPaymentContext/constants';
+import { useModalPaymentDispatch } from '../../utilities/hooks/ModalPaymentContext/context';
+import { smallScreens } from '../../utilities/general/responsive';
+import { getWebsiteImages } from '../../utilities/general/StoreImages';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import ReactPixel from 'react-facebook-pixel';
+
+const GiftMealHighlightBox = () => {
+  const websiteImages = getWebsiteImages();
+  const { t } = useTranslation();
+
+  const onButtonClick = (e: any) => {
+    ReactPixel.trackCustom('GiftMealBoxButtonClick', {});
+    // TODO: Add router to open GAM page
+  };
+
+  return (
+    <Container>
+      <Image src={websiteImages.gamHero} alt="banner" />
+
+      <ColumnContainer>
+        <Header>{t('gamBox.header')}</Header>
+        <Description>{t('gamBox.description')}</Description>
+      </ColumnContainer>
+
+      <Button className={'button--filled'} onClick={onButtonClick}>
+        {t('gamBox.button')}
+      </Button>
+    </Container>
+  );
+};
+
+export default GiftMealHighlightBox;
+
+const Container = styled.div`
```

Comment:
I feel like we're reusing these styles a lot between our other cards--any opportunities to refactor?

Reply:
There are definitely opportunities. After the new GAM experience is launched, I want to refactor this into a shared component with Donation Pool Box.